linux/GPU.c

```diff
@@ -0,0 +1,272 @@
+/*
+htop - GPU.c
+(C) 2023 htop dev team
+Released under the GNU GPLv2+, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "linux/GPU.h"
+
+#include <assert.h>
+#include <ctype.h>
+#include <dirent.h>
+#include <errno.h>
+#include <sys/types.h>
+
+#include "XUtils.h"
+
+#include "linux/LinuxMachine.h"
+
+
+#define INVALID_CLIENT_ID ((unsigned long long int)-1)
```

Comment:
What about having a `typedef` for the client type?

Reply:
client ids are only used inside this one file, so It think a typedef is not needed

Reply:
I find ulli hard to read and lacking of context, which a typedef, even local to this file, could provide.